util/keeper/errors.go

```diff
@@ -1,9 +1,13 @@
 package keeper
 
 import (
+	"errors"
+
 	"github.com/VolumeFi/whoops"
 )
 
 const (
 	ErrNotFound = whoops.Errorf("item (%T) not found in store: %s")
```

Comment:
When possible, I like to follow boyscout rules of leaving a place tidier than you found it. This would be a great opportunity to remove the `whoops` usage from this file.

Reply:
Thank you, that makes sense. I'll keep it in mind moving forward, as I get more comfortable changing things without breaking them! For now, I think it's best to merge this as is.